nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -495,12 +498,13 @@ define([
                         method: message.error_method,
                         exceptionType: message.error_type,
                     });
-                    updateState('appError');
-                    toggleTab('error');
+                    updateState('error');
+                    switchToTab('error');
```

Comment:
Ideally these would be set at the same time -- leaving this as-is for now though.